styles/main.css

```diff
@@ -0,0 +1,111 @@
+/* main section  */
```

Comment:
Reminder to create css files for each section and move the styling in each appropriate file. Remember to style for both mobile and desktop. Initial styling is for mobile and media queries for desktop or tablet(if you desire)